### DISCUSSION TITLE
Unreal Piston Engine Simulation - throttle not working

### DISCUSSION THREAD

**Discussion by rickechler** [JSBSim-Team/jsbsim#945] [Q&A] [answered]
I'm modifying the Unreal sample project to implement a warbird with a Pratt and Whitney 1830. I've used the P51d as a template and have gotten the plane roughly set up. I've exposed manifold pressure, oil temp and some other engine states to start to familiarize myself with the code and how the plugin works. I've also implemented and exposed the prop advance to control RPM.

However, the throttle doesn't seem to affect manifold pressure even though the throttle command debug seems to be registering a change. 

I did disable the autothrottle system the P51d comes with. 

Any ideas why the throttle isn't affecting manifold pressure?

Thanks!

**Comment by gallonmate** [comment, marked as answer]
> I did disable the autothrottle system the P51d comes with.

In what way did you disable it?
If you go into the p51d's propulsion file and comment out the lines below, then it will remove the autothrottle control and it will give you back manual throttle control:
https://github.com/JSBSim-Team/jsbsim/blob/8a807a7b6f59a6195c5b1c6bfc7df84f9a75ac01/aircraft/p51d/Systems/propulsion.xml#L142-L148

I looked at the p51d briefly and I'm not really sure what the original method of throttle control was supposed to be. Looking at the file history shows the p51d was originally for Flightgear and had some features removed to work with jsbsim standalone.

**Comment by rickechler** [comment]
Thank you gallonmate! Throttle is working :)
